### DISCUSSION TITLE
pls help

### DISCUSSION THREAD

**Discussion by iAmZam0** [Verwex/Funkin-Mic-d-Up-SC#61] [Q&A]
hi,I've been trying to extract the file for days and hours but when I extract it it tells me that it is damaged, well I extracted it with 7-Zip,and when entering the folder there were no resources, docs, export there was only assets art and the .github folder, I came to ask if it is the file or an error please answer

**Comment by TentaRJ** [comment]
### Please don't make multiple discussions on the same topic, as you have already made one on #60

**Reply by iAmZam0** [reply]
I don't know where it is; -; I can not find it
